graph/tags.go

```diff
@@ -462,18 +462,18 @@ func (store *TagStore) poolAdd(kind, key string) (*progressreader.Broadcaster, b
 	return broadcaster, false
 }
 
-func (store *TagStore) poolRemove(kind, key string) error {
+func (store *TagStore) poolRemove(kind, key string, broadcasterResult error) error {
```

Comment:
If we are going to have this method called with a `nil` argument everywhere, let's have a version called `poolRemoveWithError`.